docs/install.rst

```diff
@@ -161,12 +147,12 @@ Offline Install
         This documentation is for the version of Rally currently under development. We do not provide offline installation packages for development versions.
         Were you looking for the `documentation of the latest stable version <//esrally.readthedocs.io/en/stable/>`_?
 
-If you are in a corporate environment where your servers do not have any access to the Internet, you can use Rally's offline installation package. Follow these steps to install Rally:
+If you are in a corporate environment using Linux servers that do not have any access to the Internet, you can use Rally's offline installation package. Follow these steps to install Rally:
 
 1. Install all prerequisites as documented above.
 2. Download the offline installation package for the `latest release <https://github.com/elastic/rally/releases/latest>`_ and copy it to the target machine(s).
-3. Decompress the installation package with ``tar -xzf esrally-dist-*.tar.gz``.
-4. Run the install script with ``sudo ./esrally-dist-*/install.sh``.
+3. Decompress the installation package with ``tar -xzf esrally-dist-linux-*.tar.gz``.
+4. Run the install script with ``sudo ./esrally-dist-linux-*/install.sh``.
```

Comment:
The `linux-` portion of the file is not present on the release link above.

Reply:
Also, should this be `offline-install.sh` ? Ahh it looks like we sed in teh values in the offline-install for the release version, and then its just `install.sh`.

Reply:
> The `linux-` portion of the file is not present on the release link above.

(If I understand the question correctly): Yeah it's not present now, but it will be after we release in the future due the changes here